libsigner/src/events.rs

```diff
@@ -349,3 +435,86 @@ impl EventReceiver for SignerEventReceiver {
         }
     }
 }
+
+/// Process a stackerdb event from the node
+fn process_stackerdb_event(
+    local_addr: Option<SocketAddr>,
+    mut request: HttpRequest,
+    is_mainnet: bool,
+) -> Result<SignerEvent, EventError> {
+    debug!("Got stackerdb_chunks event");
+    let mut body = String::new();
+    if let Err(e) = request.as_reader().read_to_string(&mut body) {
+        error!("Failed to read body: {:?}", &e);
+
+        request
+            .respond(HttpResponse::empty(200u16))
+            .expect("response failed");
```

Comment:
I know this is copy pasta from above, but can we get rid of the expect?  Presumably network errors could cause the response to fall, so we shouldn't crash, plus we're returning an error anyway so might as well just pass on the network error?

Reply:
I can do that now :)

Reply:
Done!